command/daemon.go

```diff
@@ -68,8 +104,8 @@ func daemonCommand(cctx *cli.Context) error {
 
 	log.Infow("Shutting down daemon")
 
-	ctx, cancel := context.WithTimeout(context.Background(), shutdownTimeout)
-	defer cancel()
+	_, shCancel := context.WithTimeout(context.Background(), shutdownTimeout)
```

Comment:
should we not get rid of this entirely if the derived context is no longer used?

Reply:
I left a few placeholders there in case we needed them in  the future, but I agree is better to keep it clean and add it if we need it. Fixed.